config/itemTypes.json

```
@@ -935,6 +963,218 @@
+    "name": "Shotgun"
+  },
+  {
+    "id": "/LongGuns/PrimeCorinth/",
+    "name": "Shotgun"
+  },
+  {
+    "id": "/LongGuns/PrimeCedo/",
+    "name": "Shotgun"
+  },
+  {
+    "id": "/BoardExec/Primary/CrpBEArcaPlasmor/",
+    "name": "Shotgun"
+  },
+  {
+    "id": "/MK1Series/MK1Strun",
+    "name": "Shotgun"
+  },
+  {
+    "id": "/MK1Series/MK1Paris",
+    "name": "Bow"
+  },
+  {
+    "id": "/Bows/",
+    "name": "Bow"
+  },
+  {
+    "id": "/Bow/",
+    "name": "Bow"
+  },
+  {
+    "id": "/LongGuns/RepeatingCrossbow/",
+    "name": "Crossbow"
+  },
+  {
+    "id": "/LongGuns/PrimeZhuge/",
+    "name": "Crossbow"
+  },
+  {
+    "id": "/LongGuns/TnoPrmryXbow/",
+    "name": "Crossbow"
+  },
+  {
+    "id": "/LongGuns/GarudaCrossbow/",
+    "name": "Crossbow"
+  },
+  {
+    "id": "/LongGuns/PrimeNagantaka/",
+    "name": "Crossbow"
+  },
+  {
+    "id": "/LongGuns/InfArmCannon/",
+    "name": "Shotgun"
+  },
+  {
+    "id": "/LongGuns/CrpSplitLaser/",
+    "name": "Shotgun"
+  },
+  {
+    "id": "/LongGuns/CrpBriefcaseLauncher/",
+    "name": "Launcher"
+  },
+  {
+    "id": "/ClanTech/Chemical/RocketLauncher",
+    "name": "Launcher"
+  },
+  {
+    "id": "/ClanTech/Bio/BioWeapon",
+    "name": "Launcher"
+  },
+  {
+    "id": "/LongGuns/GrenadeLauncher/",
+    "name": "Launcher"
+  },
+  {
+    "id": "/Syndicates/PerrinSequence/LongGuns/PSPenta",
+    "name": "Launcher"
+  },
+  {
+    "id": "/LongGuns/GrnGrenadeLauncher/",
+    "name": "Launcher"
+  },
+  {
+    "id": "/KuvaLich/LongGuns/Tonkor/",
+    "name": "Launcher"
+  },
+  {
+    "id": "/KuvaLich/LongGuns/Ogris/",
+    "name": "Launcher"
+  },
+  {
+    "id": "/KuvaLich/LongGuns/Zarr/",
+    "name": "Launcher"
+  },
+  {
+    "id": "/LongGuns/GrnCannon/",
+    "name": "Launcher"
+  },
+  {
+    "id": "/LongGuns/TnHopliteSpear/",
+    "name": "Speargun"
+  },
+  {
+    "id": "/LongGuns/TnPriestSpear/",
+    "name": "Speargun"
+  },
+  {
+    "id": "/LongGuns/PrimeScourge/",
+    "name": "Speargun"
+  },
+  {
+    "id": "/LongGuns/Spears/",
+    "name": "Speargun"
+  },
+  {
+    "id": "/Grineer/LongGuns/GrnFlameSpear/",
+    "name": "Speargun"
+  },
+  {
+    "id": "/BoardExec/Primary/CrpBEFerrox/",
+    "name": "Speargun"
+  },
+  {
+    "id": "/LongGuns/CrpSentAmlgSniper/",
+    "name": "Sniper Rifle"
+  },
+  {
+    "id": "/ClanTech/Energy/Railgun",
+    "name": "Sniper Rifle"
+  },
+  {
+    "id": "/LongGuns/TnQuadSniper/",
+    "name": "Sniper Rifle"
+  },
+  {
+    "id": "/LongGuns/FiveShotSniper/",
+    "name": "Sniper Rifle"
+  },
+  {
+    "id": "/LongGuns/RubicoPrime/",
+    "name": "Sniper Rifle"
+  },
+  {
+    "id": "/Rifle/SniperRifle",
+    "name": "Sniper Rifle"
+  },
+  {
+    "id": "/Rifle/VandalSniperRifle",
+    "name": "Sniper Rifle"
+  },
+  {
+    "id": "/LongGuns/InfSniperRifle/",
+    "name": "Sniper Rifle"
+  },
+  {
+    "id": "/InfestedLich/LongGuns/CodaSporothrix",
+    "name": "Sniper Rifle"
+  },
+  {
+    "id": "/LongGuns/PrimeLightningGun/",
```

```suggestion
    "id": "/Lotus/Weapons/Tenno/LongGuns/PrimeLightningGun/PrimeLightningGun",
```
Does vadarya have scope?